src/Workspaces/CoreTestUtilities/Formatting/FormattingTestBase.cs

```diff
@@ -21,7 +21,7 @@ public abstract class FormattingTestBase
             Dictionary<OptionKey, object> changedOptionSet = null,
             bool testWithTransformation = true)
         {
-            return AssertFormatAsync(expected, code, SpecializedCollections.SingletonEnumerable(new TextSpan(0, code.Length)), language, debugMode, changedOptionSet, testWithTransformation);
+            return AssertFormatAsync(expected, code, new[] { new TextSpan(0, code.Length)}, language, debugMode, changedOptionSet, testWithTransformation);
```

Comment:
The `SpecializedCollections` type came through an IVT attribute.  Once the code was moved it was difficult to keep without causing other IVT conflicts.  Hence I just simplified and used an array.

If this is an important perf point I can work to fix this in a later check-in.